README.md

```diff
@@ -72,9 +72,9 @@ For details on all releases, refer to [CHANGELOG.md](./CHANGELOG.md).
 
 ## Parabol Core Team
 
-* [jordanh](https://github.com/jordanh)
-* [ackernaut](https://github.com/ackernaut)
-* [mattkrick](https://github.com/mattkrick)
+- [jordanh](https://github.com/jordanh)
```

Comment:
Look like my prettier settings updated this - I have no strong feelings about `-` instead of `*`!